src/EntityFramework.NuGet/EntityFramework.NuGet.csproj

```diff
@@ -2,7 +2,8 @@
   <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
 
   <PropertyGroup>
-    <TargetFrameworks>net45;net40;$(DefaultNetCoreTargetFramework)</TargetFrameworks>
+    <TargetFrameworks>net45;net40;netcoreapp3.0</TargetFrameworks>
```

Comment:
Why netcoreapp3.0?

Reply:
Well, technically, it should be netstandard2.1, but then it couldn't depend on ef6.csproj (which is also netcoreapp3.0)

The actual package's targets are specified in a custom .nuspec file.